pushToServer.sh

```diff
@@ -58,3 +64,73 @@ fi
 
 
 
+
```

Comment:
why this comment block?

Reply:
Wasn't working for me, so asked chatGPT to re-arrange it and make any corrections necessary, left it there so that you can see what I've done, but will delete :)